.github/workflows/main.yml

```diff
@@ -1,21 +1,68 @@
 name: build
-on: [push, pull_request]
+
+on:
+  pull_request:
+  push:
+  schedule:
+    # Prime the caches every Monday
+    - cron: 0 1 * * MON
+
 jobs:
-  run:
-    name: Build
-    runs-on: ${{ matrix.operating-system }}
+  unix:
     strategy:
+      fail-fast: false
       matrix:
-        operating-system: [macos-latest, ubuntu-latest, windows-latest]
-        ocaml-compiler: [ '4.04.0', '4.11.0' ]
+        os:
+          - macos-latest
```

Comment:
why the removal of windows?

Reply:
It actually existed as a separate job below, but since it turned out that 2.1 mode couldn't be used even on Unix at the moment, it was reverted to simpler one.